config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
-  resources :articles
-  root 'welcome#show'
-  devise_for :users
+  resources :articles do
+    resources :comments
+  end
+  #get 'articles#index'
+  root 'articles#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.